components/brave_new_tab_ui/components/default/footer/footer.tsx

```diff
@@ -75,13 +75,13 @@ export default class FooterInfo extends React.PureComponent<Props, {}> {
                 {getLocale('customize')}
               </IconButtonSideText>
             </IconButtonContainer>
-            <IconLink title={getLocale('preferencesPageTitle')} href='chrome://settings'>
+            <IconLink title={getLocale('preferencesPageTitle')} href='brave://settings'>
```

Comment:
This change could make link as brave url but this url is not loaded from webui page.
Brave's webui url seems to have brave scheme but it's actually chrome scheme.
That means we replace `brave` scheme with `chrome` scheme before start loading.
and webui only load allowed url because webui is previledged page.
Internally, renderer think brave and chrome are different scheme, loading brave url from webui page(chrome cheme) is blocked.

Reply:
Our previous fix was https://github.com/brave/brave-core/pull/1825. (but reverted.)
Its approach was replacing chrome scheme with brave from status bubble UI.
I think we can try this approach again.